<?xml version="1.0"?>
<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="Arm_R_end_effector" />
  <arg name="timeout" default="0.005" />

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find eod_robot_description)/urdf/eod_robot_description.xacro'" />


  <node name="trac_ik_tests" pkg="trac_ik_examples" type="ik_tests" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
